dbos-rules.js

```diff
@@ -53,9 +54,35 @@ const extConfig =
 module.exports = {
   meta: {
     "name": "@dbos-inc/eslint-plugin",
-    "version": "0.0.2",
+    "version": "0.0.4",
   },
   rules: {
+    'detect-native-code': {
```

Comment:
I was thinking we could detect all native code by checking the presence of `.c` files - and the like - in the `node_modules`. But I assume this would not be done at the `eslint` level? Maybe in the build runtime?

Reply:
Correct.

There are multiple approaches, the most thorough ones don't give you specific help early.

If you type 'bcrypt.hash' into an editor that supports eslint (and have it configured), it can tell your right then and there that you messed up, and what to do differently (specifically, use bcryptjs).

![image](https://github.com/dbos-inc/eslint-plugin/assets/134347445/5d729d2d-0ae1-4620-97e8-4b0501fdd0bc)

If you want to block all native code categorically and not be in the business of saying what to do instead and not count on the user to install eslint + care about the output, then another approach is called for.